### PR TITLE
feat: marketing component

### PR DESCRIPTION
**Pull request by gis727**
@madelyndubuk :
- added the marketing component
- added a license button in the footer
- made the "Get started button" redirect to the GitHub repository
- made the Home component the root component ( **/** )

@gis727 :
- added more description

Here is the result:
Screenshot 2020-07-01 at 6 36 43 PM

### CODE REVIEW

File: packages/frontend/src/app/home/home.component.html [hunk 1]
```diff
@@ -17,23 +17,49 @@
...
+            Find out which tests pass and fail nondeterministically.
+            <br/>
+            Easily traceback which build led to those flaky tests.
+        </p>
+    </section>
+
+    <section class="description-section">
+        <h3 class="description-header">Deploy your own flaky</h3>
+        <p class="description-text">
+            With a single click, deploy your own version of flaky.dev !
```
Comment (bcoe) [inline packages/frontend/src/app/home/home.component.html:47]: This feature sounds great, but it's probably a ways off 😆 

Perhaps we can say instead, 

"
Deploy your own flaky.dev

Flaky.dev is developed in the open on GitHub.

You can easily deploy a version to your own GCP account, or help add support for other platforms.
``

File: packages/frontend/src/app/home/home.component.html [hunk 2]
```diff
@@ -17,23 +17,49 @@
...
+            Automatically open new issues for failling tests.
+        </p>
+    </section>
+
+    <section class="description-section">
+        <h3 class="description-header">Spot flaky tests</h3>
+        <p class="description-text">
+            Find out which tests pass and fail nondeterministically.
+            <br/>
+            Easily traceback which build led to those flaky tests.
```
Comment (bcoe) [inline packages/frontend/src/app/home/home.component.html:40]: Easily identify the specific build that introduced a flaky test.

File: packages/frontend/src/app/home/home.component.html [hunk 3]
```diff
@@ -17,23 +17,49 @@
...
+            Track every test in your repositories.
+            <br/>
+            Automatically open new issues for failling tests.
+        </p>
+    </section>
+
+    <section class="description-section">
+        <h3 class="description-header">Spot flaky tests</h3>
+        <p class="description-text">
+            Find out which tests pass and fail nondeterministically.
```
Comment (bcoe) [inline packages/frontend/src/app/home/home.component.html:38]: Based on historical build information, flaky.dev calculates which of your tests exhibit flakiness.

File: packages/frontend/src/app/home/home.component.html [hunk 4]
```diff
@@ -17,23 +17,49 @@
...
-    <app-search (searchOptionSelected)="onSearchOptionSelected($event)"></app-search>
+<!--DESCRIPTIONS-->
+<div class="description">
+
+    <section class="description-section">
+        <h3 class="description-header">One place for every test</h3>
+        <p class="description-text">
+            Track every test in your repositories.
+            <br/>
+            Automatically open new issues for failling tests.
```
Comment (bcoe) [inline packages/frontend/src/app/home/home.component.html:31]: "All your tests in one place"?

"Get an organization-wide insights the test health of your repositories"?


> Automatically open new issues for failing tests.

☝️ I think we should potentially leave this out until we add the feature.

File: packages/frontend/src/app/home/home.component.html [hunk 5]
```diff
@@ -17,23 +17,49 @@
...
 </div>
 
+
+
 <!--INTRODUCTION BUTTON-->
 <div id="introduction-btn">
-    <button mat-raised-button color="primary">Get started</button>
-</div>
\ No newline at end of file
+    <a mat-raised-button color="primary" target="_blank" href="https://github.com/GoogleCloudPlatform/flaky-service">Get started on GitHub</a>
```
Comment (bcoe) [inline packages/frontend/src/app/home/home.component.html:59]: perhaps, "Follow development on GitHub"?

File: packages/frontend/src/app/home/home.component.html [hunk 6]
```diff
@@ -17,31 +17,52 @@
...
-<!--SEARCH BAR-->
-<div id="search-bar">
-    <app-search (searchOptionSelected)="onSearchOptionSelected($event)"></app-search>
+<!--DESCRIPTIONS-->
+<div class="description">
+
+    <section class="description-section">
+        <h3 class="description-header">All your tests in one place</h3>
+        <p class="description-text">
+            Get an organization-wide insights on the test health of your repositories.
```
Comment (cedpeters) [inline packages/frontend/src/app/home/home.component.html:29]: "Get organization-wide insights into the test health of your repositories" maybe?

**Comment by gis727** [comment]
@bcoe  The hamburger is being dropped in another PR (https://github.com/GoogleCloudPlatform/flaky-service/pull/46)

**Comment by gis727** [comment]
@bcoe Since the cla bot was blocking, removed the hamburger (to retrigger checks).

**Pull request merged**